ush/radmon_diag_ck.sh

```diff
@@ -71,23 +71,23 @@ echo "--> radmon_diag_ck.sh"
    #---------------------------------------------
    #  get list of diag files in the radstat file 
    #
-   radstat_contents=`tar -tf ${radstat_file} | grep '_ges' |  
+   radstat_contents=`tar -tf "${radstat_file}" | grep '_ges' |
 		gawk -F"diag_" '{print $2}' | 
 		gawk -F"_ges" '{print $1}'`
 
   
    #---------------------------------------------
    #  load contents of satype_file into an array
    #
-   satype_contents=`cat ${satype_file}`
+   satype_contents=`cat "${satype_file}"`
 
  
    #-------------------------------------------------
    #  compare $satype_contents and $radstat_contents
    #    report anything missing 
    #
-   for sat in $satype_contents; do
-     test=`echo $radstat_contents | grep $sat`
+   for sat in ${satype_contents}; do
+     test=`echo "${radstat_contents}" | grep "${sat}"`
      
      if [[ ${#test} -le 0 ]]; then
```

Comment:
For consistency with previous.
```suggestion
     content_count=$(echo "${radstat_contents}" | grep "${sat}" | wc -l)

     if (( content_count <= 0 )); then
```